.babelrc

```diff
@@ -1,5 +1,8 @@
 {
   "plugins": [
+    // Stage 0
+    "@babel/plugin-proposal-function-bind",
```

Comment:
I wouldn’t recommend including this one; it’s unlikely to progress at this time.